### PR TITLE
Update redirection regex for BadmintONTO and SemicONTO

### PR DESCRIPTION
**Pull request by huanyu-li**

### CODE REVIEW

**Comment by davidlehn** [comment]
You should test this locally.  The server logs are saying this:

```
[Tue Apr 02 16:18:04.950950 2024] [core:alert] [pid ...] [client ...] /.../w3id.org/BadmintONTO/.htaccess: RewriteRule: cannot compile regular expression '^(([0-9]\\.[0-9](?:\\.[0-9])?)/?$'
```

I've not used this tool much, but it also flags the regex error:
https://htaccess.madewithlove.com/

**Comment by dgarijo** [comment]
Exactly, we recommend to test the files locally before pushing your changes :)

**Comment by huanyu-li** [comment]
> You should test this locally. The server logs are saying this:
> 
> ```
> [Tue Apr 02 16:18:04.950950 2024] [core:alert] [pid ...] [client ...] /.../w3id.org/BadmintONTO/.htaccess: RewriteRule: cannot compile regular expression '^(([0-9]\\.[0-9](?:\\.[0-9])?)/?$'
> ```
> 
> I've not used this tool much, but it also flags the regex error: https://htaccess.madewithlove.com/

Thank you all! 
I tested my regular expression before and again and it works, and also with your recommended tool.
However, from the server log, I see the regular expression is shown slightly different from what I have in the htaccess file.
I have "^([0-9]\.[0-9](?:\.[0-9])?)/?$" but in the error log there is an extra left parentheses which makes it cannot be compiled. Could it be the reason?

**Comment by davidlehn** [comment]
Might be a typo in the turtle rule, which has that extra left paren.  apache will fail on the whole file when one rule has a syntax issue.

File: BadmintONTO/.htaccess [hunk 1]
```diff
@@ -5,15 +5,17 @@ RewriteEngine on
...
 
 # Redirect request to versioned/latest RDF/XML
 RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
-RewriteRule ^([\w\-_0-9]+)/?$ https://huanyu-li.github.io/BadmintONTO/$1/BadmintONTO.rdf [R=303,NE,L]
+RewriteRule ^([0-9]\.[0-9](?:\.[0-9])?)/?$ https://huanyu-li.github.io/BadmintONTO/$1/BadmintONTO.rdf [R=303,NE,L]
 
 # Redirect request to versioned/latest Turtle
 RewriteCond %{HTTP_ACCEPT} text/turtle
-RewriteRule ^([\w\-_0-9]+)/?$ https://huanyu-li.github.io/BadmintONTO/$1/BadmintONTO.ttl [R=303,NE,L]
+RewriteRule ^(([0-9]\.[0-9](?:\.[0-9])?)/?$ https://huanyu-li.github.io/BadmintONTO/$1/BadmintONTO.ttl [R=303,NE,L]
```
Comment (TallTed) [inline BadmintONTO/.htaccess:17]:
```suggestion
RewriteRule ^([0-9]\.[0-9](?:\.[0-9])?)/?$ https://huanyu-li.github.io/BadmintONTO/$1/BadmintONTO.ttl [R=303,NE,L]
```

**Comment by huanyu-li** [comment]
Thank you so much!

**Pull request merged**